.gitignore

```diff
@@ -1,6 +1,4 @@
 node_modules/
-dist/
-docs/
```

Comment:
Does this mean we need to commit these output files in these two folders?

Reply:
@huningx Since JamesIves/github-pages-deploy-action will automatically deploy master to gh-pages, so if we want keep dist and docs folers, we need modify .gitignore file without dist and docs, then .gitignore on gh-pages branch is right.